doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -134,6 +138,215 @@ Examine the FluentBit sidecar's STDOUT to see logs for Ray's component processes
+  name: pod-log-reader
+subjects:
+- kind: ServiceAccount
+  name: fluent-bit
+  namespace: default
+```
+
+Now, in order to make the access permissions for the Fluent Bit plugin effective, you create the role and role binding, defined above, by executing the command:
+
+```bash
+kubectl apply -f eks-fluent-bit-daemonset-rbac.yaml
+```
+
+In contrast to the ECS case, where we backed the configuration into a custom image, in our Kubernetes setup we’re using a config map to define the log parsing and routing for the Fluent Bit plugin. For this, use a file called `eks-fluent-bit-configmap.yaml` with the following content:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-bit-config
+  labels:
+    app.kubernetes.io/name: fluentbit
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Parsers_File  parsers.conf
+    [INPUT]
+        Name              tail
+        Tag               kube.*
+        Path              /var/log/containers/*.log
+        Parser            docker
+        DB                /var/log/flb_kube.db
+        Mem_Buf_Limit     5MB
+        Skip_Long_Lines   On
+        Refresh_Interval  10
+    [FILTER]
+        Name parser
+        Match **
+        Parser nginx
+        Key_Name log
+    [OUTPUT]
+        Name firehose
+        Match **
+        delivery_stream eks-stream
+        region us-west-2 
+  parsers.conf: |
+    [PARSER]
+        Name   nginx
+        Format regex
+```
+
+Create this config map
+
+```bash
+kubectl apply -f eks-fluent-bit-configmap.yaml
+```
+
+Define the Kubernetes Daemonset (using said config map) in a file called `eks-fluent-bit-daemonset.yaml` with below content:
+
+```yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentbit
+  labels:
+    app.kubernetes.io/name: fluentbit
+spec:
+  selector:
+    matchLabels:
+      name: fluentbit
+  template:
+    metadata:
+      labels:
+        name: fluentbit
+    spec:
+      serviceAccountName: fluent-bit
+      containers:
+      - name: aws-for-fluent-bit
+        image: amazon/aws-for-fluent-bit:1.2.0
+        volumeMounts:
+        - name: varlog
+          mountPath: /var/log
+        - name: varlibdockercontainers
+          mountPath: /var/lib/docker/containers
+          readOnly: true
+        - name: fluent-bit-config
+          mountPath: /fluent-bit/etc/
+        - name: mnt
+          mountPath: /mnt
+          readOnly: true
+        resources:
+          limits:
+            memory: 500Mi
+          requests:
+            cpu: 500m
+            memory: 100Mi
+      volumes:
+      - name: varlog
+        hostPath:
+          path: /var/log
+      - name: varlibdockercontainers
+        hostPath:
+          path: /var/lib/docker/containers
+      - name: fluent-bit-config
+        configMap:
+          name: fluent-bit-config
+      - name: mnt
+        hostPath:
+          path: /mnt
+```
+
+Finally, launch the Fluent Bit daemonset:
+```bash
+kubectl apply -f eks-fluent-bit-daemonset.yaml
+```
+
+
```

Comment:
It would be nice to have a final step here in the tutorial which the user can run, and which clearly "proves" that the log forwarding worked. But not sure how much work it is to add